app/scripts/modules/amazon/src/loadBalancer/configure/common/targetGroupValidators.ts

```diff
@@ -0,0 +1,58 @@
+import { get } from 'lodash';
+import { Validators, robotToHuman, spelNumberCheck } from '@spinnaker/core';
+import { ITargetGroup } from 'amazon/domain';
+
+export const isNameInUse = (
+  existingNames: { [account: string]: { [region: string]: string[] } },
+  credentials: string,
+  region: string,
+) => (name: string) =>
+  get(existingNames, [credentials, region], []).includes(name.toLowerCase())
+    ? `There is already a target group in ${credentials}:${region} with that name.`
+    : null;
+
+export const isNameLong = (appNameLength: number) => (name: string) =>
+  name.length < 32 - appNameLength
+    ? null
+    : 'Target group name is automatically prefixed with the application name and cannot exceed 32 characters in length.';
+
+export const isDuplicateName = (duplicateGroups: string[]) => (name: string) =>
```

Comment:
This validator already exists in core as `valueUnique`

Reply:
Great, will convert to that!